src/pkg/artifact/dao/model.go

```diff
@@ -67,9 +65,10 @@ func (t *Tag) TableName() string {
 
 // ArtifactReference records the child artifact referenced by parent artifact
 type ArtifactReference struct {
-	ID       int64 `orm:"pk;auto;column(id)"`
-	ParentID int64 `orm:"column(parent_id)"`
-	ChildID  int64 `orm:"column(child_id)"`
+	ID       int64  `orm:"pk;auto;column(id)"`
+	ParentID int64  `orm:"column(parent_id)"`
+	ChildID  int64  `orm:"column(child_id)"`
+	Platform string `orm:"column(platform)"`
```

Comment:
A string can cover the platform related info? I remembered there were other information related to the `platform`

Reply:
It's the database model, the platform is a json string